e2e/keywords/common.resource

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation     
+...               Keywords defined here can be used where this Keywords.resource in loaded.
+Library    ../libs/steps.py
+Resource    ../keywords/utility.resource
+
+*** Variable ***
+${retry_time_out}=    600
+${retry_interval}=    5
+
+*** Keywords ***
+Volume state should be ${volume_expect_state}
+    Run keyword And Continue On Failure    Wait Until Keyword Succeeds    ${retry_time_out} seconds    ${retry_interval} seconds    Check volume state should be ${volume_expect_state}
+
+Node ${node_number} with replica node state should be ${node_expect_state}
+    ${target_node_number} =    Evaluate    ${node_number}-1
+    Run keyword And Continue On Failure    Wait Until Keyword Succeeds    ${retry_time_out} seconds    ${retry_interval} seconds    Check node ${node_number_with_replica}[${target_node_number}] state should be ${node_expect_state}
+
+Node ${node_number} without replica node state should be ${node_expect_state}
```

Comment:
Suggest consolidating to `Node ${node_number} state should be ${expected_node_state}`:
- Node ${node_number} with replica node state should be ${node_expect_state}
- Node ${node_number} without replica node state should be ${node_expect_state}

Reply:
Per [#1300 (comment)](https://github.com/longhorn/longhorn-tests/pull/1300#discussion_r1270475922) mentioned, I 've consolidated the duplicating keywords. Thank you.